04_Running_The_Class.Rmd

```diff
@@ -0,0 +1,285 @@
+
+:::: {.borrowed_chunk}
+```{r, echo = FALSE, results='asis'}
+cow::borrow_chapter(
+  doc_path = "child/_child_terra_group_add_member.Rmd",
+  repo_name = "jhudsl/AnVIL_Template"
+)
+```
+::::
+
+## Have Students Create Accounts
+
+Each of your students will need to:
+
+1. Create a Google account, if they do not already have one
+2. Sign in to Terra
+3. Provide you with their username, so that you can add them to your Student Group
+
+Student-facing instructions for account creation (and several other activities) are provided in the [**Student Instructions**](#student-instructions) section.
+
+## Add Students to Group
+
+Once your students have accounts, you will need to add them to your Student Group as **Members**.
+
+::: {.notice}
+In order to add students to your Group, you will need the usernames for the Google accounts your students are using to access AnVIL.
+:::
+
+Unfortunately there is currently no way to add members to a Group in bulk, so you will need to add students to the Group one-by-one.
+
+The following instructions explain how to add someone to a Terra Group.  You will want to add students to the Group as **Members**.
+
+:::: {.borrowed_chunk}
+```{r, echo = FALSE, results='asis'}
+cow::borrow_chapter(
+  doc_path = "child/_child_terra_group_add_member.Rmd",
+  repo_name = "jhudsl/AnVIL_Template"
+)
+```
+::::
+
+## Add Student Group to Workspace(s)
+
+Now that you have added students to the Group, you can easily share Terra resources with the entire Group.
+
+First, you will need to give your Student Group access to any Workspaces that they will need.
+
+If you are **using a Public Workspace** (most pre-made content uses Public Workspaces), **you can skip this step** - Public Workspaces are automatically available to everyone on AnVIL.  
+
+If you have **Workspaces of your own** that your students will need to access, you should **add the Student Group to your Workspace as a *Reader***.
+
+- Adding students as a Reader gives them permission to view the Workspace, and (once they have funding) to **clone their own copy**.  This will enable them to work safely in their own space without interfering with each other or overwriting anything in the original Workspace.
+- Adding students to your Workspace(s) can safely be done ahead of time; as Readers they can't run computations and run up a bill.
+
+To add the Student Group to the Workspace, you will need the Group email address, which can be found on the Group management page: https://anvil.terra.bio/#groups
+
+The following instructions explain how to add someone to a Terra Workspace.  You will want to add students to the Workspace as **Readers**.
+
+:::: {.borrowed_chunk}
+```{r, echo = FALSE, results='asis'}
+cow::borrow_chapter(
+  doc_path = "child/_child_workspace_share.Rmd",
+  repo_name = "jhudsl/AnVIL_Template"
+)
+```
+::::
+
+Make sure the newly added Student Group displays "Reader" under "Roles".
+
+# Teach the Class
```

Comment:
Might want to create some spacing here - looks a bit weird with the Large Dropcap. <img width="756" alt="image" src="https://user-images.githubusercontent.com/15618412/229290673-549a969e-3c61-4539-b202-fcd400c0c4ee.png">

Reply:
@avahoffman what browser were you using, and does it look better now?  I added an extra `<br>`

I'm using chrome and it looked okay to me (not shifted over like your picture), so it's hard for me to know if I fixed it.

Reply:
Looks good! Apparently I'm the only person who uses Firefox, so feel free to ignore :)

<img width="782" alt="image" src="https://user-images.githubusercontent.com/15618412/229635817-25301ac5-e3c3-4d98-9897-1b6b9857fbbb.png">